internal/providers/file/file.go

```diff
@@ -31,6 +32,9 @@ const (
 
 func FetchConfig(f resource.Fetcher) (types.Config, report.Report, error) {
 	filename := os.Getenv(cfgFilenameEnvVar)
+	if f := flag.Lookup("validate"); f != nil && f.Value.String() != "" {
```

Comment:
This feels gross (using flag.Lookup), but the alternatives are changing the interfaces to allow passing more information to FetchConfig, using a global, or changing the enviroment variables.

Reply:
Ditto.